neo/SmartContract/InteropService.cs

```diff
@@ -552,6 +555,9 @@ private static bool Contract_Call(ApplicationEngine engine)
             }
 
             ExecutionContext context_new = engine.LoadScript(contract.Script, 1);
+            context_new.GetState<ExecutionContextState>().ReadOnly = currentManifest != null &&
+                (currentManifest.ReadOnlyMethods.IsWildcard || currentManifest.ReadOnlyMethods.Contains(methodStr));
```

Comment:
If we remove the null check this line will be easier to read

Reply:
This was wrong, is from `contract`, not `currentManifest`